iothub_client/src/iothub_client_ll_uploadtoblob_uhttp.c

```diff
@@ -590,7 +584,13 @@ static int IoTHubClient_LL_UploadToBlob_step3(IOTHUB_CLIENT_LL_UPLOADTOBLOB_HAND
             case IOTHUB_CREDENTIAL_TYPE_X509_ECC:
             case IOTHUB_CREDENTIAL_TYPE_DEVICE_AUTH:
             {
-                if (send_http_request(upload_data, http_client, STRING_c_str(relativePathNotification), request_header, messageBody) != 0)
+                // Codes_SRS_IOTHUBCLIENT_LL_02_065: [ If creating the HTTPAPIEX_HANDLE fails then IoTHubClient_LL_UploadMultipleBlocksToBlob(Ex) shall fail and return IOTHUB_CLIENT_ERROR. ]
+                if ((http_client = create_http_client(upload_data)) == NULL)
```

Comment:
Isn't the client being created in another step already? Why does it need to be created here?

Reply:
Good catch! No, it shouldn't need to be created again here.